plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/preferences/IOpenShiftCoreConstants.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.core.preferences;
+
+/**
+ * @author jeff.cantrill
+ */
+public interface IOpenShiftCoreConstants {
+	
+	static final String OPENSHIFT_CLI_LOC = "openshift.cli.location"; 
```

Comment:
Shouldnt we have this implemented in OpenShiftCorePreferences? Isnt this a constant for the v3 preferences?